notebooks/ch14/bias_modeling.md

```diff
@@ -216,14 +252,22 @@ plt.title('Irreducible error');
 ```
 
 
-![png](bias_modeling_files/bias_modeling_20_0.png)
+![png](bias_modeling_files/bias_modeling_22_0.png)
 
 
 ## Bias-Variance In Practice
 
-In practice, we do not know the population data generation process and thus are unable to precisely determine a model's bias, variance, or irreducible error. Instead, we use our observed dataset as an approximation to the population. As we have seen, however, achieving a low training error does not necessarily mean that our model will have a low test error as well. Fundamentally, this occurs because training error reflects the bias of our model but not the variance; the test error reflects both.
+In practice, we do not know the population data generation process and thus are unable to precisely determine a model's bias, variance, or irreducible error. Instead, we use our observed dataset as an approximation to the population. 
+
+As we have seen, however, achieving a low training error does not necessarily mean that our model will have a low test error as well. Fundamentally, this occurs because training error reflects the bias of our model but not the variance; the test error reflects both. In order to minimize test error, our model needs to simultaneously achieve low bias and low variance. There is always a always a trade-off between these two values. It is easy to obtain a model with extremely low bias but high variance (by fitting a curve that passes through every training observation), or a model with very low variance but high bias (by fitting a horizontal line to the data). The challenge lies in finding a model for which both the variance and the  bias are low.
```

Comment:
I think combining this with:
"Underfitting is typically caused by too much bias; overfitting is typically caused by too much variance."
in the takeaways section would be nice.

Reply:
Restructuring this section a bit but also think leaving "Underfitting is typically caused by too much bias; overfitting is typically caused by too much variance." in the takeaways section as a summary

Reply:
Sure, seems fine to me.